tests/state/test_stateful_count_policy.py

```diff
@@ -0,0 +1,69 @@
+import time
+from datetime import datetime
+
+import pytest
+from arroyo import Message, Partition, Topic
+from arroyo.backends.kafka.consumer import KafkaPayload
+from arroyo.processing.strategies.dead_letter_queue.policies.abstract import (
+    InvalidMessage,
+)
+
+from snuba.redis import redis_client
+from snuba.state.stateful_count import StatefulCountInvalidMessagePolicy
+
+NAME = "consumer-group"
+SAVED_NAME = f"dlq:{NAME}"
+INVALID_MESSAGE_EXCEPTION = InvalidMessage(
+    Message(Partition(Topic(""), 0), 0, KafkaPayload(None, b"", []), datetime.now())
+)
+
+
+class TestStatefulCountPolicy:
+    @pytest.fixture
+    def policy(self) -> StatefulCountInvalidMessagePolicy[KafkaPayload]:
+        return StatefulCountInvalidMessagePolicy(
+            consumer_group_name=NAME, limit=5, seconds=1
+        )
+
+    @pytest.fixture(autouse=True)
+    def _clear_hash(self) -> None:
+        redis_client.delete(SAVED_NAME)
+
+    def test_simple(
+        self, policy: StatefulCountInvalidMessagePolicy[KafkaPayload]
+    ) -> None:
+        now = int(time.time())
+        policy.handle_invalid_message(INVALID_MESSAGE_EXCEPTION)
+        assert redis_client.hgetall(SAVED_NAME) == {str(now).encode("utf-8"): b"1"}
+
+    def test_past_limit(
+        self, policy: StatefulCountInvalidMessagePolicy[KafkaPayload],
+    ) -> None:
+        # limit is 5, should raise on 6th invalid message
+        for _ in range(5):
+            policy.handle_invalid_message(INVALID_MESSAGE_EXCEPTION)
+        with pytest.raises(InvalidMessage):
+            policy.handle_invalid_message(INVALID_MESSAGE_EXCEPTION)
```

Comment:
This tests only covers the functionality of the the base class. I think you should at least add an assertion about the redis state.